core/jvm/src/main/scala/zio/sql/expr.scala

```diff
@@ -108,7 +108,11 @@ trait ExprModule extends NewtypesModule with FeaturesModule with OpsModule {
 
     def typeTagOf[A](expr: Expr[_, _, A]): TypeTag[A] = expr.asInstanceOf[InvariantExpr[_, _, A]].typeTag
 
-    implicit def literal[A: TypeTag](a: A): Expr[Features.Literal, Any, A] = Expr.Literal(a)
+    implicit def literal[A](a: A)(implicit typeTag: TypeTag[A]): Expr[Features.Literal, Any, A] =
+      typeTag match {
+        case TypeTag.TString => Expr.Literal(s"'${a.toString}'").asInstanceOf[Expr[Features.Literal, Any, A]]
```

Comment:
This is how String should be interpreted/rendered, so it a better fit to do this operation in PostgresModule#renderRead